packages/andive/src/components/direction/destination.tsx

```diff
@@ -62,16 +72,14 @@ const DestinationRoad = styled(({offsetY, ...props}) => <div {...props} />)`
   background: ${palette.darkPrimary};
 `
 
-const AsideLabel = styled(({offsetY, ...props}) => <div {...props} />)`
+const AsideLabel = styled(({...props}) => <div {...props} />)`
```

Comment:
A remplacter par `styled.div` du coup, vu qu'on a plus besoin d'extraire un prop spécifique. On extrait des props avec cette syntaxe pour ne pas qu'ils se retrouvent sur le tag HTML in fine.